custom/enikshay/tasks.py

```diff
@@ -63,32 +73,36 @@ def run(self):
         update_count = 0
         noupdate_count = 0
         error_count = 0
-        for episode in self._get_open_episode_cases():
-            adherence_update = EpisodeAdherenceUpdate(episode, self)
-            voucher_update = EpisodeVoucherUpdate(self.domain, episode)
-            try:
-                update_json = adherence_update.update_json()['update']
-                update_json.update(voucher_update.update_json())
-                case_block = self._get_case_block(update_json, episode.case_id)
-                if case_block:
-                    submit_case_blocks(
-                        [ElementTree.tostring(case_block.as_xml())],
-                        self.domain
+        with Timer() as t:
+            for episode in self._get_open_episode_cases():
+                adherence_update = EpisodeAdherenceUpdate(episode, self)
+                voucher_update = EpisodeVoucherUpdate(self.domain, episode)
+                try:
+                    update_json = adherence_update.update_json()['update']
+                    update_json.update(voucher_update.update_json())
+                    case_block = self._get_case_block(update_json, episode.case_id)
+                    if case_block:
+                        submit_case_blocks(
+                            [ElementTree.tostring(case_block.as_xml())],
+                            self.domain
+                        )
+                        update_count += 1
+                    else:
+                        noupdate_count += 1
+                except Exception, e:
+                    error_count += 1
+                    logger.error(
+                        "Error calculating updates for episode case_id({}): {}".format(
+                            episode.case_id,
+                            e
+                        )
                     )
-                    update_count += 1
-                else:
-                    noupdate_count += 1
-            except Exception, e:
-                error_count += 1
-                logger.error(
-                    "Error calculating updates for episode case_id({}): {}".format(
-                        episode.case_id,
-                        e
-                    )
-                )
         logger.info(
-            "Summary of enikshay_task: Updated {updates}, errored on {errors} and {noupdates} "
-            "cases didn't need update ".format(updates=update_count, errors=error_count, noupdates=noupdate_count)
+            "Summary of enikshay_task: domain: {domain}, duration (sec): {duration} "
```

Comment:
This is run for only adherence_task [toggle enabled domains](https://github.com/dimagi/commcare-hq/blob/73c71b6cc753c0ea949596f290fbd6d650a158ab/custom/enikshay/tasks.py#L42), there are couple of enikshay test domains that will also need to have this run